recipe/activate.bat

```diff
@@ -1,3 +1,20 @@
+@@echo off
+setlocal enabledelayedexpansion
+
+:: save existing variables for deactivation script; initialize all backups to a placeholder
+:: (empty is the same as undefined), and only overwrite them if a value exists
+for %%X in (
+    CC CXX CMAKE_ARGS CMAKE_GENERATOR CMAKE_GENERATOR_PLATFORM CMAKE_GENERATOR_TOOLSET
+    CMAKE_PREFIX_PATH CONDA_BUILD_CROSS_COMPILATION DISTUTILS_USE_SDK INCLUDE
+    LIB MSSdk MSYS2_ARG_CONV_EXCL MSYS2_ENV_CONV_EXCL PY_VCRUNTIME_REDIST
+    VS_MAJOR VS_VERSION VS_YEAR
+) do (
+    set "_CONDA_BACKUP_%%X=placeholder"
```

Comment:
Remove this line?

Reply:
This is part of the logic to properly unset values upon activation. Otherwise, if one does
```batch
set "VS_YEAR=%_CONDA_BACKUP_VS_YEAR%"
```
for a variable that wasn't set before activation (and thus an unset `_CONDA_BACKUP_FOO`), it's possible to end up with the literal string `"%_CONDA_BACKUP_VS_YEAR%"` after deactivation.

Reply:
Why can't we just use `if defined !_CONDA_BACKUP_%%X!` in deactivate?